frontend/src/metabase/dashboard/components/DashboardInfoSidebar/DashboardInfoSidebar.tsx

```diff
@@ -136,48 +183,103 @@ const DashboardInfoSidebarBody = ({
           hasError={!!descriptionError}
           placeholder={t`Add description`}
           key={`dashboard-description-${dashboard.description}`}
-          style={{ fontSize: ".875rem" }}
+          className={DashboardInfoSidebarS.EditableDescription}
         />
         {!!descriptionError && (
           <Text color="error" size="xs" mt="xs">
             {descriptionError}
           </Text>
         )}
-      </ContentSection>
-
-      {!dashboard.archived && (
-        <ContentSection>
-          <Stack spacing="md">
-            <Switch
-              disabled={!canWrite}
-              label={t`Auto-apply filters`}
-              labelPosition="left"
-              variant="stretch"
-              size="sm"
-              id={autoApplyFilterToggleId}
-              checked={dashboard.auto_apply_filters}
-              onChange={e => handleToggleAutoApplyFilters(e.target.checked)}
+      </SidesheetCard>
+      <SidesheetCard title={t`Last edit`}>
+        <Flex lh="1" align="center" gap=".25rem">
+          <Icon name="pencil" />
+          {c(
+            "This phrase describes when a dashboard was last edited, and by whom. {1} is the date. {0} is the name of the editor.",
+          ).jt`${(
+            <>
+              <DateTime unit="day" value={lastEditDate} />
+            </>
+          )} by ${lastEditor}`}
+        </Flex>
+      </SidesheetCard>
+      {dashboard.entity_id && (
+        <SidesheetCard
+          title={
+            <Group spacing="sm">
+              {t`Entity ID`}
+              <Popover position="top">
+                <Popover.Target>
+                  <Icon
+                    name="info"
+                    cursor="pointer"
+                    style={{ position: "relative", top: "-1px" }}
+                  />
+                </Popover.Target>
+                <Popover.Dropdown>
+                  <Paper p="md" maw="12.5rem">
+                    {t`When using serialization, replace the sequential ID with this global entity ID to have stable URLs across environments. Also useful when troubleshooting serialization.`}{" "}
+                    {showMetabaseLinks && (
+                      <>
+                        <Link
+                          target="_new"
+                          to="https://www.metabase.com/docs/latest/installation-and-operation/serialization"
```

Comment:
this link is not correct, we need to interpolate the name. fortunately there's a shiny new `useDocsUrl` hook that will give you the correct url and the `showMetabaseLinks` var

see: https://github.com/metabase/metabase/pull/47433